lib/editor/components/EditorInput.js

```diff
@@ -118,6 +118,28 @@ export default class EditorInput extends Component<Props> {
     }
   }
 
+  _renderRouteTypeOptions = (field: GtfsSpecField): React.Node => {
+    const options = field.options || []
+    const toOption = ({value, text}) => (
+      <option key={value}>{text} ({value})</option>
+    )
+    // The first 10 options (types 0 - 12) are the standard types found in the
+    // GTFS specification. All others represent extended route types.
+    const routeTypes = [
+      {
+        label: 'Standard',
+        options: options.slice(0, 9).map(toOption)
```

Comment:
Using a filter on the range for `value` may be easier to read. As written, it looks like the array indexes and the comment above contradict each other.
```suggestion
        options: options.filter(opt => opt.value < 100).map(toOption)
```